packages/db/src/schema/core/defs.ts

```diff
@@ -248,10 +248,40 @@ export const admins = pgTable(
   ],
 );
 
+/**
+ * Arenas are grouping mechanisms for organizing related competitions
+ * Stores metadata and classification for discovery/filtering
+ */
+export const arenas = pgTable(
+  "arenas",
+  {
+    id: text().primaryKey().notNull(),
+    name: text().notNull(),
+    createdBy: text("created_by"),
+    classification: jsonb().notNull(),
```

Comment:
Seems like we know exactly what information is stored in here. Is there any reason to use jsonb? Can we just have `category` and `skill` columns?

Reply:
Still wondering what the answer is here.

Reply:
Apologies - missed this earlier - altered in [0d4142b](https://github.com/recallnet/js-recall/pull/1531/commits/0d4142b9ae76cf624daaeabae592b3a936615893)